serde_derive/Cargo.toml

```diff
@@ -26,7 +26,7 @@ proc-macro = true
 [dependencies]
 proc-macro2 = "1.0"
 quote = "1.0"
-syn = { version = "1.0", features = ["visit"] }
+syn = { version = "1.0", features = ["visit", "extra-traits"] }
```

Comment:
"extra-traits" is pretty expensive in compile time. Please avoid introducing a dependency on it.

Reply:
I replaced the use of `==` on `syn::AttrStyle` with a structural match.